src/main/bash/sdkman-install.sh

```diff
@@ -77,26 +77,35 @@ function __sdkman_install_candidate_version {
 }
 
 function __sdkman_install_local_version {
-	local candidate version folder
+	local candidate version folder versionLength versionLengthMax
```

Comment:
It's just a little cosmetic thing but could you make the new variables snake case please?